modules/atom/pom.xml

```diff
@@ -206,7 +206,7 @@
         </dependency>
         <dependency>
             <groupId>org.apache.activemq</groupId>
-            <artifactId>activemq-core</artifactId>
+            <artifactId>activemq-broker</artifactId>
```

Comment:
what's the difference between broker and core? is it possible that some user might be depending on some funcionality that is on core and now is no longer provided by the broker? if so, at the very minimum there should be a migration guide entry